conf/settings/estimation/body_to_imu.xml

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE settings SYSTEM "../settings.dtd">
 
-<settings>
+<settings target="ap|nps">
```

Comment:
Because it fails with sim target where imu.c is not compiled. If you want to keep it without a specific target, I suggest to fix sim then.